.github/workflows/release.yml

```diff
@@ -70,11 +69,6 @@ jobs:
           components: rustfmt
           profile: minimal
 
-      - uses: Swatinem/rust-cache@v2
-        with:
-          prefix-key: appflowy-lib-cache
-          key: ${{ matrix.job.os }}-${{ matrix.job.target }}
-
```

Comment:
Are these changes intentional?

Reply:
Yes, we shouldn't use cache when building the release package, as it may encounter errors.